Update behavior with project color shortcut entry in DD button (#4545)
qgis/QGIS-Documentation: source/docs/user_manual/introduction/general_tools.rst

* pick a color from anywhere on your computer display
* choose a color from the color selector dialog

.. note:: When the color widget is set to a :ref:`project color <project_colors>`

>   through the data-defined override properties, the above functions allowing to

Reviewer: ```suggestion
  through the data-defined override properties, the above functions for
```

>   change the color are unavailable. You'd first need to :guilabel:`Unlink color`

Reviewer: ```suggestion
  changing the color are unavailable. You'd first need to :guilabel:`Unlink color`
```
